doc/source/ad_dp2.rst

```diff
@@ -0,0 +1,237 @@
+          command: bin/rest-server.sh
+          memory: 1G
+          disk_quota: 2G
+          timeout: 180
+          instances: 1
+        services:
+        - bryn-cdh
+        - <YOUR_POSTGRESQL_SERVICE_NAME_HERE> for example "pg-atk-ebi"
+        - bryn-zk
+        env:
+          CC_URI: <CLOUD_FOUNDRY_API_ENDPOINT> 
+          UAA_URI: <UAA_ENDPOINT> 
+          UAA_CLIENT_NAME: atk-client
+          UAA_CLIENT_PASSWORD: c1oudc0w
+
+#)  Create an instance of PostgreSQL by running the command: 
+
+    .. code::
+
+        $ cf create-service postgresql93 free pg-atk-ebi
+
+    and you should see an output like this:
+
+    .. code::
+
+        Creating service instance pg-atk-ebi in org seedorg / space seedspace as admin...
+        OK
+
+#)  Change to the "~/vcap/app" folder (or wherever you have
+    "trustedanalytics.tar.gz" unpacked).
+#)  Now run the command ``cf push``.
+    This takes a few minutes to run and you should see the following output:
+
+    .. code::
+
+        [hadoop@master app]$ cf push
+        Using manifest file /home/hadoop/vcap/app/manifest.yaml
+        Creating app atk-ebi in org seedorg / space seedspace as admin...
+        OK
+        Using route atk-ebi.apps.gotapaas.eu
+        Binding atk-ebi.apps.gotapaas.eu to atk-ebi...
+        OK
+        Uploading atk-ebi...
+        Uploading app files from: /home/hadoop/vcap/app
+        Uploading 48.3K, 9 files
+        Done uploading
+        OK
+        Binding service bryn-cdh to app atk-ebi in org seedorg / space seedspace as admin...
+        OK
+        Binding service pg-atk-ebi to app atk-ebi in org seedorg / space seedspace as admin...
+        OK
+        Binding service bryn-zk to app atk-ebi in org seedorg / space seedspace as admin...
+        OK
+        Starting app atk-ebi in org seedorg / space seedspace as admin...
+        0 of 1 instances running, 1 starting
+        1 of 1 instances running
+        App started
+
+        OK
+        App atk-ebi was started using this command `bin/rest-server.sh`
+        Showing health and status for app atk-ebi in org seedorg / space seedspace as admin...
+        OK
+        requested state: started
+        instances: 1/1
+        usage: 1G x 1 instances
+        urls: atk-ebi.apps.gotapaas.eu
+        last uploaded: Wed May 20 22:22:54 UTC 2015
+        stack: cflinuxfs2
+        state since cpu memory disk details
+        #0 running 2015-05-20 03:25:13 PM 0.0% 622.9M of 1G 432.9M of 2G
+
+    If you like to see the complete configuration for your app, run the
+    command "cf env atk-ebi".
+#)  Retrieve data from VCAP_APPLICATION uris.
+#)  Create a client credentials file.
+    For more information,
+    see https://github.com/trustedanalytics/atk/wiki/python-client
+#)  To tail your app logs:
+
+    .. code::
+       
+        cf logs atk-ebi
+
+#)  Open a Python2.7 or IPython session and do the following:
+
+    .. code::
+
+        In [1]: import trustedanalytics as atk
```

Comment:
Here and other places, we should "import trustedanalytics as ta"

not "as atk"